config.toml

```diff
@@ -1,138 +1,241 @@
-navbar = "https://raw.githubusercontent.com/cncf/artwork/master/projects/etcd/icon/color/etcd-icon-color.png"
-
-[params.social]
-github = "https://github.com/etcd-io/etcd"
-
-[[params.features.primary]]
-title = "Simple interface"
-svg = "interface"
-description = "Read and write values using standard HTTP tools, such as curl"
-
-[[params.features.primary]]
-title = "Key-value storage"
-svg = "kv"
-description = "Store data in hierarchically organized directories, as in a standard filesystem"
-
-[[params.features.primary]]
-title = "Watch for changes"
-svg = "watch"
-description = "Watch specific keys or directories for changes and react to changes in values"
-
-[[params.features.secondary]]
-text = "Optional SSL client certificate authentication"
-icon = "lock"
-
-[[params.features.secondary]]
-text = "Benchmarked at 1000s of writes/s per instance"
-icon = "chart-bar"
-
-[[params.features.secondary]]
-text = "Optional TTLs for keys expiration"
-icon = "clock"
-
-[[params.features.secondary]]
-text = "Properly distributed via Raft protocol"
-icon = "arrows-alt"
-
-[[params.fonts]]
-name = "Source Sans Pro"
-sizes = [300,400,600,700]
-
-[[params.fonts]]
-name = "Fira Mono"
-sizes = [300,400,600,700]
-
-[[params.adopters]]
-logo = "https://raw.githubusercontent.com/cncf/artwork/master/projects/kubernetes/horizontal/color/kubernetes-horizontal-color.png"
-link = "https://kubernetes.io"
-description = "etcd is the backend for service discovery and stores cluster state and configuration"
-
-[[params.adopters]]
-logo = "https://raw.githubusercontent.com/cncf/artwork/master/projects/rook/horizontal/color/rook-horizontal-color.png"
-link = "https://rook.io"
-description = "etcd serves as the orchestration engine for Rook"
-
-[[params.adopters]]
-name = "Patroni"
-link = "https://github.com/zalando/patroni"
-description = "A template for PostgreSQL HA with ZooKeeper, etcd, or Consul"
-
-[[params.adopters]]
-name = "Trillian"
-link = "https://github.com/google/trillian/"
-description = "A transparent, highly scalable and cryptographically verifiable data store, created by Google"
-
-[[params.adopters]]
-logo = "https://raw.githubusercontent.com/cncf/artwork/master/projects/coredns/horizontal/color/coredns-horizontal-color.png"
-link = "https://coredns.io"
-description = "CoreDNS uses etcd as an optional backend"
-
-[[params.adopters]]
-logo = "/img/m3-logo.png"
-link = "https://eng.uber.com/m3/"
-description = "M3, a large-scale metrics platform for Prometheus created by Uber, uses etcd for rule storage and other functions"
-
-[[params.adopters]]
-link = "https://openstack.org"
-logo = "/img/openstack-logo.svg"
-description = "OpenStack supports etcd as an optional provider of configuration storage, distributed key locking, and more"
-
-[mediaTypes."text/netlify"]
-delimiter = ""
-
-[outputFormats.REDIRECTS]
-mediaType = "text/netlify"
-baseName = "_redirects"
-
-[outputs]
-home = ["HTML", "REDIRECTS"]
```

Comment:
If you remove the redirects config then the _redirects-file rules will stop working. Not what we want.

Reply:
Ah, so reinstate lines 137 & 138

Reply:
Is this fixed by uncommenting:
https://github.com/etcd-io/website/pull/112/files#diff-28043ff911f28a5cb5742f7638363546311225a63eabc365af5356c70d4deb77R235-R240

Reply:
Reinstate lines 130 to 138

Reply:
Uncommented lines 232-240.